rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -1703,6 +1703,13 @@ boolean restartTask(@HeaderParam("sessionid") String sessionId, @PathParam("jobi
 			UnknownTaskRestException, PermissionRestException;
 
 	@PUT
+	@Path("jobs/{jobid}/tasks/{taskname}/finishInErrorTask")
+	@Produces("application/json")
+	boolean finishInErrorTask(@HeaderParam("sessionid") String sessionId, @PathParam("jobid") String jobid,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: org.ow2.proactive_grid_cloud_portal.scheduler.exception.NotConnectedRestException, org.ow2.proactive_grid_cloud_portal.scheduler.exception.UnknownJobRestException, org.ow2.proactive_grid_cloud_portal.scheduler.exception.UnknownTaskRestException, org.ow2.proactive_grid_cloud_portal.scheduler.exception.PermissionRestException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1160)